units/src/fee.rs

```diff
@@ -14,6 +14,7 @@
 use core::ops;
 
 use NumOpResult as R;
+use crate::NumOpError as E;
```

Comment:
nit: The alias is only used in one place. Personally I think the code is more clear if this type is imported directly and used in the error return statement.

I'll ack still.

Reply:
see https://github.com/rust-bitcoin/rust-bitcoin/pull/4480